Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -221,6 +221,17 @@ public class LocalStorage {
         }
     }
 
+    static func moveToSubmittedJobs(jobId: String) throws {
+        try createDirectory(at: submittedDirectory, overwrite: false)
+        let sourceURLs = try fileManager.contentsOfDirectory(atPath: jobId).map {
+            URL(fileURLWithPath: jobId).appendingPathComponent($0)
+        }
+        for sourceURL in sourceURLs {
```

Comment:
instead of moving each file over individually, we can just move the entire jobId directory

Reply:
actually this was marked as draft because I had a different approach that I had not pushed